### PR TITLE
RANCHER-545 create scheduled build of Scheduled BugFest Cypress Tests…

### PR DESCRIPTION
**Pull request by OHaimanov**
… pipeline

### CODE REVIEW

**Comment by penfriend** [comment]
@OHaimanov please add also parameters related to testrail. Thank you!

File: pipelines/tests/cypress/cypressBugFestScheduledPipeline.groovy [hunk 1]
```diff
@@ -38,8 +37,9 @@ pipeline {
                         string(name: 'tenant', value: tenant),
                         string(name: 'user', value: 'folio-aqa'),
                         password(name: 'password', value: 'Folio-aqa1'),
-                        string(name: 'cypressParameters', value: "--env grepTags=smoke,grepFilterSpecs=true"),
-                        string(name: 'customBuildName', value: JOB_BASE_NAME)
+                        string(name: 'cypressParameters', value: "--env grepTags=\"smoke criticalPth\",grepFilterSpecs=true"),
+                        string(name: 'customBuildName', value: JOB_BASE_NAME),
+                        string(name: 'timeout', value: '6')
```
Comment (penfriend) [inline pipelines/tests/cypress/cypressBugFestScheduledPipeline.groovy:42]: please add these parameters:
 testrailProjectID: 14
testrailRunID: 2099

Comment (OHaimanov) [reply pipelines/tests/cypress/cypressBugFestScheduledPipeline.groovy]: done

File: pipelines/tests/cypress/cypressBugFestScheduledPipeline.groovy [hunk 2]
```diff
@@ -38,8 +37,9 @@ pipeline {
                         string(name: 'tenant', value: tenant),
```
[the remark this answers is not in the document]
Comment (OHaimanov) [reply pipelines/tests/cypress/cypressBugFestScheduledPipeline.groovy:37]: done

**Review by penfriend — APPROVED** [review]

**Pull request merged**